Helper/LuceneSyntaxToBddSyntax.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace OpenOrchestra\MongoBundle;
+
+use OpenOrchestra\Helper\LuceneSyntaxToBddSyntaxInterface;
+
+/**
+ * Class LuceneSyntaxToBddSyntax
+ */
+class LuceneSyntaxToBddSyntax implements LuceneSyntaxToBddSyntaxInterface
+{
+	CONST OPERATOR = '+- ';
+	CONST RESERVED = '+- )(';
+	CONST DELIMITER = '##';
```

Comment:
Maybe you could put this const inthe interface.
Or are they bind to this specific implementation ?